internal/utils/logger.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 Red Hat, Inc.
```

Comment:
I'm surprised we need to have this file at all. Isn't there a logging library that we can use instead?